requirements.txt

```diff
@@ -26,8 +26,8 @@ protobuf==5.27.3
 pyasn1==0.5.1
 pyasn1-modules==0.3.0
 pycparser==2.22
-pydantic==2.9.1
-pydantic-core==2.23.3
+pydantic==2.9.2
+pydantic-core==2.23.4
```

Comment:
@williballenthin can we remove pydantic-core from this file, updates seem to always fail and this appears redundant?!

Reply:
since this is pinned by pydantic, seems fine to me